bscript/interpreter/README.md

```diff
@@ -0,0 +1,42 @@
+interpreter
+========
+
+[![ISC License](http://img.shields.io/badge/license-ISC-blue.svg)](http://copyfree.org)
+[![GoDoc](https://pkg.go.dev/badge/github.com/libsv/go-bt/bscript/interpreter?utm_source=godoc)](http://godoc.org/github.com/libsv/got-bt/bscript/interpreter)
+
+Package interpreter implements the an interpreter for the bitcoin transaction language.  There is
+a comprehensive test suite.
+
+This package has intentionally been designed so it can be used as a standalone
+package for any projects needing to use or validate bitcoin transaction scripts.
+
+## Bitcoin Scripts
+
+Bitcoinsv provides a stack-based, FORTH-like language for the scripts in
+the bitcoin transactions.  This language is not turing complete
+although it is still fairly powerful.  A description of the language
+can be found at https://wiki.bitcoinsv.io/index.php/Script
+
+## Installation and Updating
+
+```bash
+$ go get -u github.com/libsv/go-bt/bscript/interpreter
+```
+
+## Examples
+
+* [Standard Pay-to-pubkey-hash Script](http://github.com/libsv/go-bt/bscript/interpreter#example-PayToAddrScript)  
```

Comment:
where should these links go to?

Reply:
Hmmm not sure, I'll look into this today